app/services/scheduled_reporter.rb

```diff
@@ -0,0 +1,56 @@
+class ScheduledReporter
+  attr_reader :check_time
+
+  def initialize(check_time)
```

Comment:
I didn't assume we would always use current time. I.e., the clockwork job misfires and we need to manually run all reports for a given day. Also makes testing more explicit (no need for Timecop).